Modix.Bot/Modules/AuthorizationModule.cs

```diff
@@ -17,85 +18,84 @@ namespace Modix.Modules
     [HelpTags("claims")]
     public class AuthorizationModule : ModuleBase
     {
+        private readonly IAuthorizationService _authorizationService;
+
         public AuthorizationModule(IAuthorizationService authorizationService)
         {
-            AuthorizationService = authorizationService;
+            _authorizationService = authorizationService;
         }
 
         [Command("claims")]
-        [Summary("Lists the currently assigned claims for the current user, or a given user.")]
+        [Summary("Lists the currently assigned claims for the calling user, or given user")]
         public async Task ClaimsAsync(
-            [Summary("The user for whom to list claims, if any.")]
-                IGuildUser guildUser = null)
+
+            [Summary("User whom to list the claims for, if any")]
+            IGuildUser user = null)
+
         {
-            var claims = (guildUser == null)
-                    ? AuthorizationService.CurrentClaims
-                    : await AuthorizationService.GetGuildUserClaimsAsync(guildUser);
+            // If a user was provided, get their claims, otherwise get the caller's claims.
+            var claims = user is null
+                ? _authorizationService.CurrentClaims
+                : await _authorizationService.GetGuildUserClaimsAsync(user);
 
-            await ReplyWithClaimsAsync(claims);
+            await ReplyWithClaimsAsync(claims.ToList());
```

Comment:
Useless memory allocation.

Reply:
This stops multiple enumeration from `.Any()` then `.Select(x =>)`, which I'd imagine is worse(?)

Reply:
It should be fine actually - `_authorizationService.CurrentClaims` is a static array.

Reply:
This method also counts for the case they specify a user, however. Not a huge deal though.

Reply:
`.Any()` should check whether the object has a `.Count` property, and will check that instead of enumerating, if it does.